ci/main.go

```diff
@@ -0,0 +1,211 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"dagger.io/dagger"
+)
+
+const (
+	API_VERSION = "v1"
+
+	Go2WASM   = "go1.21"
+	GO2HOST   = "go1.20.4"
+	GO2PLUGIN = "go1.20.4"
+
+	// EXTRA ARGS FOR BUILDING (placed after the "go build")
+	// use -x for more details from a go compiler
+	EXTRA_HOST_ARGS = ""
+
+	SYSCALL_CLIENT_SIDE = "apiwasm/syscall/*.go"
+
+	REP = "g/file/" + API_VERSION + "/file.pb.go"
+)
+
+var (
+	// protobuf files
+	apiProto  string
+	testProto string
+
+	// protoc plugin
+	template    string
+	eneratorSrc string
+)
+
+func main() {
+	if err := build(context.Background()); err != nil {
+		log.Fatalf("Cannot build dagger pipeline %v\n", err)
+	}
+}
+
+func build(ctx context.Context) error {
+	fmt.Println("Building with Dagger")
+
+	// initialize Dagger client
+	// set the current(root) directory on the host as the working directory
+	client, err := dagger.Connect(ctx, dagger.WithLogOutput(os.Stderr))
```

Comment:
Just curious, why Stderr not Stdout?  Not a problem, just curious.

Reply:
Good question. The answer is I followed the dagger tutorial. I think it's for diagnostic purposes.

Actually, they use both. Some code snippets use Stderr and some use Stdout. Do you have any suggestions at this point? Which one is better to use here?

Reply:
I don't think there is one that is better.  I tend to use stdout most of the time because pipes work better in the shell with it,  but others don't.  I think the go standard library prints things to stderr by default.  Or maybe it's the standard logging stuff.